Lib/lib2to3/Grammar.txt

```diff
@@ -38,13 +19,13 @@ async_funcdef: 'async' funcdef
 funcdef: 'def' NAME parameters ['->' test] ':' suite
 parameters: '(' [typedargslist] ')'
 typedargslist: ((tfpdef ['=' test] ',')*
-                ('*' [tname] (',' tname ['=' test])* [',' '**' tname] | '**' tname)
+                ('*' [tname] (',' tname ['=' test])* [',' ['**' tname [',']]] | '**' tname [','])
                 | tfpdef ['=' test] (',' tfpdef ['=' test])* [','])
```

Comment:
Could it be just `tfpdef ['=' test] [',']`? Of course may be there are reasons of writing the rule as it is written due to the parser limitations.

Reply:
Maybe but I want to keep this as close to Grammar/Grammar and this is how it's voiced there.